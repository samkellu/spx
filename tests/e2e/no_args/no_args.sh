#! /bin/bash
RED='\033[0;31m'
GREEN='\033[0;34m'
YELLOW='\033[0;33m'
NC='\033[0m'

test_name="no_args"

./spx_exchange | tee tests/E2E/$test_name/$test_name.test

diff tests/E2E/$test_name/$test_name.out tests/E2E/$test_name/$test_name.test && echo -e "${GREEN}Test $test_name passed.${NC}" || echo -e "${RED}Test $test_name failed.${NC}"

gcov ./spx_exchange

rm tests/E2E/$test_name/$test_name.test
