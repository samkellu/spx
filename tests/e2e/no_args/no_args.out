[SPX] Invalid command line arguments
